AGPs/AGP-Comprehensive-Aragon-Bug-Bounty-Program.md

```diff
@@ -0,0 +1,48 @@
+---
+AGP: N/A
+Title: Comprehensive Aragon Bug Bounty Program
+Author: burrrata (@burrrata)
+Status: Stage IV
+Track: Association
+Created: 2019-10-02
+---
+
+# AGP-X: Comprehensive Aragon Bug Bounty Program
+
+## Description of desired Association policy change
+
+This proposal is to expand the [Aragon Association Bug Bounty Program](https://wiki.aragon.org/association/security/bug_bounty/) to make all Aragon projects funded by the AA via AGPs, the Nest program, or the Flock program eligible. This definition of "funded by the AA" is left broad enough to cover a range of projects from the Aragon Court to Aragon app developed through the Nest program. This does not automatically include all such project, but makes them eligible for the bug bounty program. Projects will only be accepted into the bug bounty program if they have undergone a security audit, incorporated all major and critical security audit recommendations, and shipped to mainnet. If projects make major changes after incorporating security audits and shipping to mainnet, a new security audit will be required for the projects continued eligibility in the bug bounty program. This ensures that only projects that have been audited and incorporated critical and major security audit recommendations are included. Failure to undergo a security audit or to incorporate a security team's critical and major recommendations will result in projects being dropped from the Aragon Association Bug Bounty Program.
+
+In addition to expanding the scope of the [Aragon Association Bug Bounty Program](https://wiki.aragon.org/association/security/bug_bounty/), the amounts will also be changed.
```

Comment:
```suggestion
In addition to expanding the scope of the [Aragon Association Bug Bounty Program](https://wiki.aragon.org/association/security/bug_bounty/), the amounts will also be changed. All amounts are in USD unless explicitly stated otherwise.
```